Gemfile.lock

```diff
@@ -1,6 +1,6 @@
 GIT
   remote: git://github.com/europeana/europeana-i18n-ruby.git
-  revision: c97ef134285b7ff6a0dd55fc40673320fec8dfbc
+  revision: 400ad7ec082850d44fba2c6752ff19f726e562d7
```

Comment:
This change does nothing: https://github.com/europeana/europeana-i18n-ruby/compare/c97ef134285b7ff6a0dd55fc40673320fec8dfbc...400ad7ec082850d44fba2c6752ff19f726e562d7

Let's just leave the Gem at ref c97ef134285b7ff6a0dd55fc40673320fec8dfbc